code/game/machinery/computer/abnormality_work.dm

```diff
@@ -152,6 +152,10 @@
 		if(do_after(user, work_speed, src))
 			if(!CheckStatus(user))
 				break
+			user.remove_status_effect(/datum/status_effect/interventionshield)
+			user.remove_status_effect(/datum/status_effect/interventionshield/white)
+			user.remove_status_effect(/datum/status_effect/interventionshield/black)
+			user.remove_status_effect(/datum/status_effect/interventionshield/pale)
```

Comment:
Might want to test it, but I think putting the parent type will handle all the subtypes.

Reply:
I do remember testing this and it only removing red type when only one is said to be removed. It doesn't seem to remove the parent.